<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 127);
background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_status">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 199, 199);</string>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="temp_graph_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 199, 199);</string>
    </property>
    <property name="text">
     <string>TEMPERATURE PLOT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="humidity_graph_pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 199, 199);</string>
    </property>
    <property name="text">
     <string>HUMIDITY PLOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_15">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="temp_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 237);
background-color: rgb(200, 199, 199);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="humidity_user_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="celsius_faren">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 199, 199);</string>
    </property>
    <property name="text">
     <string>C/F</string>
    </property>
   </widget>
   <widget class="QLabel" name="threshold_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="main_display">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="temperature_heading">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_heading">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperature_threshold_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_threshold_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_heading">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
